diff -aurN gthumb-2.4.0.orig/data/gthumb.desktop.in gthumb-2.4.0/data/gthumb.desktop.in
--- gthumb-2.4.0.orig/data/gthumb.desktop.in	2004-05-28 16:08:45.000000000 +0200
+++ gthumb-2.4.0/data/gthumb.desktop.in	2004-06-12 11:59:17.347781584 +0200
@@ -2,7 +2,7 @@
 _Name=gThumb Image Viewer
 _GenericName=
 _Comment=View and organize your images
-Categories=Application;Graphics;
+Categories=Application;Graphics;Viewer;
 Encoding=UTF-8
 Exec=gthumb
 Icon=gthumb.png
