--- gthumb-3.12.2/extensions/raw_files/main.c.orig	2022-04-03 20:38:59.000000000 +0200
+++ gthumb-3.12.2/extensions/raw_files/main.c	2023-03-27 06:25:00.752465430 +0200
@@ -30,6 +30,12 @@
 #include "gth-metadata-provider-raw.h"
 
 
+#if LIBRAW_COMPILE_CHECK_VERSION_NOTLESS(0, 21)
+#  define GTHUMB_LIBRAW_INIT_OPTIONS (LIBRAW_OPIONS_NO_DATAERR_CALLBACK)
+#else
+#  define GTHUMB_LIBRAW_INIT_OPTIONS (LIBRAW_OPIONS_NO_MEMERR_CALLBACK | LIBRAW_OPIONS_NO_DATAERR_CALLBACK)
+#endif
+
 typedef enum {
 	RAW_OUTPUT_COLOR_RAW = 0,
 	RAW_OUTPUT_COLOR_SRGB = 1,
@@ -213,7 +219,7 @@ _cairo_image_surface_create_from_raw (GI
 	size_t         size;
 	GthImage      *image = NULL;
 
-	raw_data = libraw_init (LIBRAW_OPIONS_NO_MEMERR_CALLBACK | LIBRAW_OPIONS_NO_DATAERR_CALLBACK);
+	raw_data = libraw_init (GTHUMB_LIBRAW_INIT_OPTIONS);
 	if (raw_data == NULL) {
 		_libraw_set_gerror (error, errno);
 		goto fatal_error;
@@ -300,7 +306,7 @@ _cairo_image_surface_create_from_raw (GI
 		if ((original_width != NULL) && (original_height != NULL)) {
 			libraw_close (raw_data);
 
-			raw_data = libraw_init (LIBRAW_OPIONS_NO_MEMERR_CALLBACK | LIBRAW_OPIONS_NO_DATAERR_CALLBACK);
+			raw_data = libraw_init (GTHUMB_LIBRAW_INIT_OPTIONS);
 			if (raw_data == NULL)
 				goto fatal_error;
 
--- gthumb-3.12.2/extensions/raw_files/gth-metadata-provider-raw.c.orig	2022-04-03 20:38:59.000000000 +0200
+++ gthumb-3.12.2/extensions/raw_files/gth-metadata-provider-raw.c	2023-03-27 06:26:26.692045645 +0200
@@ -26,6 +26,12 @@
 #include "main.h"
 
 
+#if LIBRAW_COMPILE_CHECK_VERSION_NOTLESS(0, 21)
+#  define GTHUMB_LIBRAW_INIT_OPTIONS (LIBRAW_OPIONS_NO_DATAERR_CALLBACK)
+#else
+#  define GTHUMB_LIBRAW_INIT_OPTIONS (LIBRAW_OPIONS_NO_MEMERR_CALLBACK | LIBRAW_OPIONS_NO_DATAERR_CALLBACK)
+#endif
+
 G_DEFINE_TYPE (GthMetadataProviderRaw, gth_metadata_provider_raw, GTH_TYPE_METADATA_PROVIDER)
 
 
@@ -62,7 +68,7 @@ gth_metadata_provider_raw_read (GthMetad
 	if (!_g_mime_type_is_raw (gth_file_data_get_mime_type (file_data)))
 		return;
 
-	raw_data = libraw_init (LIBRAW_OPIONS_NO_MEMERR_CALLBACK | LIBRAW_OPIONS_NO_DATAERR_CALLBACK);
+	raw_data = libraw_init (GTHUMB_LIBRAW_INIT_OPTIONS);
 	if (raw_data == NULL)
 		goto fatal_error;
 
