diff -aurN gthumb-2.7.8.orig/libgthumb/Makefile.am gthumb-2.7.8/libgthumb/Makefile.am
--- gthumb-2.7.8.orig/libgthumb/Makefile.am	2006-05-08 18:58:14.000000000 +0200
+++ gthumb-2.7.8/libgthumb/Makefile.am	2006-08-12 18:42:57.724134689 +0200
@@ -130,7 +130,8 @@
 
 libgthumb_la_LIBADD =		\
 	$(TIFF_LIBS)		\
-	$(GTHUMB_LIBS)
+	$(GTHUMB_LIBS)		\
+	$(IPTCDATA_LIBS)
 
 gthumb-marshal.h: gthumb-marshal.list $(GLIB_GENMARSHAL)
 	$(GLIB_GENMARSHAL) gthumb-marshal.list --header --prefix=gthumb_marshal > $@
