diff -aurN gthumb-2.4.0.orig/configure.in gthumb-2.4.0/configure.in
--- gthumb-2.4.0.orig/configure.in	2004-05-28 16:08:45.000000000 +0200
+++ gthumb-2.4.0/configure.in	2004-05-29 17:22:25.988226376 +0200
@@ -230,7 +230,7 @@
 AC_PATH_PROG(GLIB_GENMARSHAL, glib-genmarshal)
 AC_PATH_PROG(GLIB_MKENUMS, glib-mkenums)
 
-ALL_LINGUAS="am az be ca cs da de en_CA en_GB es eu fi fr ga hr hu it ja ko lt lv mk ml ms nl no pl pt pt_BR ru sk sq sr sr@Latn sv uk zh_CN zh_TW"
+ALL_LINGUAS="am az be ca cs da de en_CA en_GB es eu fi fr ga hr hu it ja ko lt lv mk ml ms nb nl pl pt pt_BR ru sk sq sr sr@Latn sv uk zh_CN zh_TW"
 GETTEXT_PACKAGE=gthumb
 AC_SUBST(GETTEXT_PACKAGE)
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE, "$GETTEXT_PACKAGE", [GetText Package])
