diff -uNpr gthumb-2.7.2.orig/src/dlg-jpegtran.c gthumb-2.7.2/src/dlg-jpegtran.c
--- gthumb-2.7.2.orig/src/dlg-jpegtran.c	2006-01-04 12:21:56.000000000 +0100
+++ gthumb-2.7.2/src/dlg-jpegtran.c	2006-01-05 09:37:32.000000000 +0100
@@ -34,10 +34,10 @@
 #include <glade/glade.h>
 
 #ifdef HAVE_LIBEXIF
-#include <exif-data.h>
-#include <exif-content.h>
-#include <exif-entry.h>
-#include <exif-utils.h>
+#include <libexif/exif-data.h>
+#include <libexif/exif-content.h>
+#include <libexif/exif-entry.h>
+#include <libexif/exif-utils.h>
 #include "jpegutils/jpeg-data.h"
 #endif /* HAVE_LIBEXIF */
 
diff -uNpr gthumb-2.7.2.orig/src/gth-exif-data-viewer.c gthumb-2.7.2/src/gth-exif-data-viewer.c
--- gthumb-2.7.2.orig/src/gth-exif-data-viewer.c	2006-01-04 12:21:57.000000000 +0100
+++ gthumb-2.7.2/src/gth-exif-data-viewer.c	2006-01-05 09:37:32.000000000 +0100
@@ -37,9 +37,9 @@
 #include "image-viewer.h"
 
 #ifdef HAVE_LIBEXIF
-#include <exif-data.h>
-#include <exif-content.h>
-#include <exif-entry.h>
+#include <libexif/exif-data.h>
+#include <libexif/exif-content.h>
+#include <libexif/exif-entry.h>
 #endif /* HAVE_LIBEXIF */
 
 
diff -uNpr gthumb-2.7.2.orig/src/gth-exif-utils.h gthumb-2.7.2/src/gth-exif-utils.h
--- gthumb-2.7.2.orig/src/gth-exif-utils.h	2006-01-04 12:21:57.000000000 +0100
+++ gthumb-2.7.2/src/gth-exif-utils.h	2006-01-05 09:37:32.000000000 +0100
@@ -28,10 +28,10 @@
 
 #ifdef HAVE_LIBEXIF
 
-#include <exif-data.h>
-#include <exif-content.h>
-#include <exif-entry.h>
-#include <exif-utils.h>
+#include <libexif/exif-data.h>
+#include <libexif/exif-content.h>
+#include <libexif/exif-entry.h>
+#include <libexif/exif-utils.h>
 
 
 typedef enum { /*< skip >*/
