--- gthumb-3.12.3/gthumb/gth-browser.c.orig	2023-08-27 21:11:47.000000000 +0200
+++ gthumb-3.12.3/gthumb/gth-browser.c	2023-08-30 22:16:01.779607449 +0200
@@ -4453,9 +4453,11 @@ browser_key_press_cb (GthBrowser  *brows
 		break;
 
 	case GTH_BROWSER_PAGE_BROWSER:
+		{
 		GtkWidget *focus_widget = gtk_window_get_focus (GTK_WINDOW (browser));
 		if (! GTK_IS_ENTRY (focus_widget) && ! GTK_IS_TREE_VIEW (focus_widget))
 			result = gth_browser_file_list_key_press_cb (browser, event);
+		}
 		break;
 
 	default:
