diff -aurN gthumb-2.7.3.orig/data/gthumb.desktop.in gthumb-2.7.3/data/gthumb.desktop.in
--- gthumb-2.7.3.orig/data/gthumb.desktop.in	2006-02-13 22:19:52.000000000 +0100
+++ gthumb-2.7.3/data/gthumb.desktop.in	2006-02-14 09:05:16.714204288 +0100
@@ -2,7 +2,7 @@
 _Name=gThumb Image Viewer
 _GenericName=Image Viewer
 _Comment=View and organize your images
-Categories=GNOME;GTK;Graphics;Viewer;RasterGraphics;2DGraphics;Photography;
+Categories=GTK;GNOME;Graphics;Viewer;
 Exec=gthumb %U
 Icon=gthumb
